### PR TITLE
PT-1303 add new fields to PA

### PR DESCRIPTION
**Pull request by franck-fourel**
- Add eventType and eventID
- inputTime will be added later once validated as part of https://github.com/mdblp/platform/pull/51

### CODE REVIEW

File: data/types/activity/physical/physical.go
Comment (yvon-dblg) [inline data/types/activity/physical/physical.go]: The inputTime field is documented in commit 694e5ac1ff57b4b8d5d6e684702cf495df38d239
Why it is not implemented ?
If not needed anymore/for now, a more explicit comment is needed.

**Review by yvon-dblg — CHANGES_REQUESTED** [review]
I have one comment/question, plus this remark not fully related to this PR:
I didn't check the PR for the spec, but all examples provided the timezoneOffset only, not the timezone.
The offset can be known from the timezone, the opposite is not true. In fact, the offset is not really needed.
We should provide example with the timezone information to not breaks what we have done on this point before.

File: data/types/activity/physical/physical.go
Comment (franck-fourel) [reply data/types/activity/physical/physical.go]: inputTime is included as common object in #51 
That's the reason why it's not there. Waiting for approval of #51 to include it.

File: data/types/activity/physical/physical.go
Comment (yvon-dblg) [inline data/types/activity/physical/physical.go]: The documentation should mention this behavior:
eventId is mandatory when the eventType is specified.

File: data/types/activity/physical/physical.go
Comment (yvon-dblg) [inline data/types/activity/physical/physical.go]: The duration should be mandatory in this case

**Review by yvon-dblg — CHANGES_REQUESTED** [review]
2 questions, one I think must be change.

File: data/types/activity/physical/physical.go
Comment (franck-fourel) [reply data/types/activity/physical/physical.go]: that's correct

File: data/types/activity/physical/physical.go
Comment (franck-fourel) [reply data/types/activity/physical/physical.go]: makes sense

**Review by yvon-dblg — APPROVED** [review]
LGTM

File: docs/UpdatedData.md
Comment (CedricPerotto) [inline docs/UpdatedData.md]:
```suggestion
In the below example, The 2 objects are sent in 2 separate requests. The first object coming with the first request indicates that the physical activity is entered on the handset at 8:00am. It starts at 8:50am for 60 minutes. The second object that is received later on as part of a second request says that the duration of the same activity has been changed to 50 minutes. This last information was entered at 10:00am. This second object will become the effective one while first one can be considered as the history of changes. The link between both is done through the common `eventId`. 
```

File: docs/UpdatedData.md
Comment (CedricPerotto) [inline docs/UpdatedData.md]:
```suggestion
- `eventId`: unique ID provided by the client that is used to link stop and start events. If we receive several objects with the same ID, the most recent one will be the effective one while the other objects will be considered as history of changes. The duration is __mandatory__ when this field is provided.
```

**Review by CedricPerotto — CHANGES_REQUESTED** [review]
LGTM, just typos in the doc.
@yvon-dblg ?

File: data/types/common/eventtype.go
Comment (yvon-dblg) [inline data/types/common/eventtype.go]: Do we keep theses files ?

**Review by yvon-dblg — COMMENTED** [review]
Still working :)

File: data/types/common/eventtype.go
Comment (franck-fourel) [reply data/types/common/eventtype.go]: deleted as part of f6df691

**Comment by franck-fourel** [comment]
@yvon-dblg  ready for final review

**Review by yvon-dblg — APPROVED** [review]
LGTM
@CedricPerotto ?

**Review by CedricPerotto — APPROVED** [review]

**Pull request merged**
